package.json

```diff
@@ -2,15 +2,24 @@
   "name": "graphql-hooks",
   "version": "3.1.0",
   "description": "Graphql Hooks",
-  "main": "src/index.js",
+  "main": "lib/graphql-hooks.js",
+  "module": "es/graphql-hooks.js",
+  "unpkg": "dist/graphql-hooks.min.js",
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "prettier": "pretty-quick",
     "contributors:add": "all-contributors add",
-    "contributors:generate": "all-contributors generate"
+    "contributors:generate": "all-contributors generate",
+    "build": "rollup -c",
+    "prepublishOnly": "npm run build"
```

Comment:
We need to run `npm run build` in `prepack` also.

Reply:
If we add it in `prepack` I believe it will run twice when we run `npm publish`. Are we using `npm pack`?

Reply:
I think `npm pack` might be an edge case and if people want to use it they could do `npm run build` first?